File: GP_Inheritance/src/main/java/org/example/WbLib.java
package org.example;

public class WbLib
{
    public void Logo()
    {
        System.out.println("PROGRAM START");
    }

    public void Menu()
    {
        //"\n1. 생성 2. 삭제 3.   선택 4.       입금 5. 출금 6. 이체 7. 출력 0. 종료"
        System.out.println("[1] Create Account");   //계좌 생성
        System.out.println("[2] Delete Account");   //계좌 삭제
        System.out.println("[3] Select Account");   //계좌번호로 검색해서 번호랑 잔액 출력
        System.out.println("-------------------");
        System.out.println("[4] Input Account");    //입금
        System.out.println("[5] Output Account");   //출금
        System.out.println("[6] Send Money");        //이체
        System.out.println("-------------------");
        System.out.println("[7] Print All Accounts");  // 전체 계좌 출력
        System.out.println("-------------------");
        System.out.println("[0] Program End");
    }

    public void Ending()
    {
        System.out.println("PROGRAM END");
    }



}

File: 0604_test/0604_test/WbLib.cs
using System;

namespace _0604_test
{
    public class WbLib
    {
        public void Pause()
        {
            Console.WriteLine("\n\n엔터키를 눌러 계속");
            Console.ReadKey(true);
        }

        public void Logo()
        {
            Console.WriteLine("------------------");
            Console.WriteLine("0604");
            Console.WriteLine("관프");
            Console.WriteLine("------------------");
        }
        
        public ConsoleKey Menu()
        {
            Console.WriteLine("[F1] - 데이터 추가");
            Console.WriteLine("[F2] - 데이터 삭제");
            Console.WriteLine("[F3] - 데이터 검색");
            Console.WriteLine("[F4] - 데이터 변경");
            Console.WriteLine("[F5] - 데이터 전체출력\n");
            Console.WriteLine("[ESC] - 프로그램 종료\n");
            return Console.ReadKey().Key;
        }

        public void Ending()
        {
            Console.WriteLine("------------------");
            Console.WriteLine("끝");
            Console.WriteLine("------------------");            
        }
    }
}
